cleanup/azure.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+run_az_deletion_command() {
+  # Check the DRYRUN environment variable
+  if [ "${DRYRUN:-true}" = "false" ] || [ "${DRYRUN:-true}" = "no" ]
+  then
+    # Execute command "as it"
+    echo "command that would be executed : "
+    echo "az group" "$@"
+    az group "$@"
+  else
+    # Execute command with the "--dry-run"
+    subcommand="$1"
+    shift
+    echo "== DRY RUN (show instead of ${subcommand}):"
+    echo "az group show $@"
+    az group show "$@" 2>&1 | grep -v 'Request would have succeeded' || true
```

Comment:
```suggestion
    az group show "$@" || true
```

not needed since the grep was an assertion based on the `aws` CLI

Reply:
note needed anymore by the way it should handle the dry run now